crates/wasmtime/src/runtime.rs

```diff
@@ -690,6 +696,7 @@ pub enum ProfilingStrategy {
 #[derive(Default, Clone)]
 pub struct Engine {
     config: Arc<Config>,
+    stack_map_registry: Arc<StackMapRegistry>,
```

Comment:
I think I may be missing something, but how come this is stored in an `Engine`? (vs a `Store`)

Reply:
It was unnecessary, and is now removed.